flink-connector-opensearch/src/main/java/org/apache/flink/connector/opensearch/sink/OpensearchSinkBuilder.java

```diff
@@ -148,6 +150,7 @@ public OpensearchSinkBuilder<IN> setBulkFlushMaxActions(int numMaxActions) {
      * @param maxSizeMb the maximum size of buffered actions, in mb.
      * @return this builder
      */
+    @SuppressWarnings("UnusedReturnValue")
```

Comment:
I don't understand why these suppression are needed?

Reply:
I put it because compiler gave me a warning about it. I can remove that suppression if you prefer.

Reply:
Please remove, thank you

Reply:
Done

Reply:
Hi @reta , please let me know what are the plans for merging.

Reply:
> Hi @reta , please let me know what are the plans for merging.

Thanks @lilyevsky , LGTM, asked Flink committers to take I look (I do not have commit rights)

Reply:
Thanks @reta